core/src/createTransaction.ts

```diff
@@ -148,3 +90,78 @@ export async function createTransaction(
 
     return transaction;
 }
+
+async function createSystemInstruction(
+    connection: Connection,
+    payer: PublicKey,
+    recipient: PublicKey,
+    amount: BigNumber
+): Promise<TransactionInstruction> {
```

Comment:
Am I right that the newly supported use cases for things like bidirectional NFTs would require the server to build a `Transaction` without using `createTransaction`, or to add extra instructions after creating a transaction using it? If so, would it be worth exporting these `createSystemInstruction` and `createSPLTokenInstruction` functions as building blocks that can be used to make creating complex transactions a bit easier?

Reply:
In terms of usage, I think it's binary -- you're either using a transfer request, and the SDK provides an opinionated framework, or you're using a transaction request, and the implementation is entirely up to you.

Patterns will emerge, but should be encouraged outside of the SDK. It's only intended to be a reference implementation of the specification, and the spec doesn't care how you create arbitrary transactions.